File: Solutions in Java/TwoSum.java
class Solution {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> hm = new HashMap<Integer, Integer>();
		for(int i = 0; i < nums.length; i++){
			int sec = target - nums[i];
			if(hm.containsKey(sec)){
                return new int[]{hm.get(sec), i};
            }
			hm.put(nums[i], i);
		}
		return null;
    }
}

File: Solutions in Javascript/TwoSum.js
/**
 * @author : Anmol Agrawal
 */
/**
 * @param {number[]} nums
 * @param {number} target
 * @return {number[]}
 */
var twoSum = function(nums, target) {
    let map = new Map();
    for(let i = 0; i < nums.length; i++){
        let sec = target - nums[i];
        if(map.has(sec)){
            let ans = [];
            ans[0] = map.get(sec);
            ans[1] = i;
            return ans;
        }
        map.set(nums[i], i);
    }
    return null;
};